WordPressComKit/Me.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public class Me {
+    var ID: Int!
+    var displayName: String?
+    var username: String!
+    var email: String?
```

Comment:
Probably not although the API documentation isn't terribly clear if all return fields are non-optional or if they aren't marked with "optionality". https://developer.wordpress.com/docs/api/1.1/get/me/